HW_4/HW_4.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import os
+import time
+from pathlib import Path
+import requests
+import threading
+import multiprocessing
+import asyncio
+
+image_urls = []
+with open('images.txt', 'r') as images:
+    for image in images.readlines():
+        image_urls.append(image.strip())
+
+image_path = Path('./images/')
+
```

Comment:
Не плохо было бы проверить наличие папки
```suggestion
if not image_path.exists():
    image_path.mkdir()
```